sonar-plugin-api/src/main/java/org/sonar/api/server/rule/DefaultRuleDescriptionSection.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.api.server.rule;
+
+import java.io.IOException;
+import java.net.URL;
+import org.apache.commons.io.IOUtils;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class DefaultRuleDescriptionSection implements RuleDescriptionSection {
```

Comment:
wouldn't it be better to have this class in the package `org.sonar.api.server.rule.internal`? It's not part of the API and should never be used (casted) by plugins at runtime.

Reply:
What you suggest make sense, the implementation should be hidden from the consumers. The problem is that the builder will need to remain in that package as it will be used by plugins.

So what I propose is to make `DefaultRuleDescriptionSection` package-private and move the builder into its own public class. WDYT?

Reply:
sounds good!

Reply:
done :)